assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -239,6 +242,19 @@ const Checkout = ( { attributes, scrollToTop } ) => {
 								onChange={ setEmail }
 								required={ true }
 							/>
+							{ CHECKOUT_ALLOWS_SIGNUP && (
```

Comment:
Just pointing out this does not cover the use case when signup is a requirement. In which case, no input is needed, but the checkout API should create an account if the requester is logged out.

Reply:
Agreed, I still need to handle that case. Thanks for reviewing, I appreciate the early feedback :)